darts/tests/test_NBEATS.py

```diff
@@ -45,8 +46,16 @@ def test_fit(self):
             self.assertEqual(len(pred3), 1)
 
         def test_multivariate(self):
-            series_multivariate = tg.linear_timeseries(length=100).stack(tg.linear_timeseries(length=100))
-            model = NBEATSModel(input_chunk_length=1, output_chunk_length=1,
-                                n_epochs=10, num_stacks=1, num_blocks=1, layer_widths=20)
-            with self.assertRaises(ValueError):
-                model.fit(series_multivariate)
+
+            # testing a 2-variate linear ts, first one from 0 to 1, second one from 0 to 0.5, length 100
+            series_multivariate = tg.linear_timeseries(length=100).stack(tg.linear_timeseries(length=100, start_value = 0, end_value=0.5))
+            model = NBEATSModel(input_chunk_length=1, output_chunk_length=1, n_epochs=20)
```

Comment:
how about maybe test a slightly less trivial version of the model here? For instance with `input_chunk_length=3` or something like that? I fear that values of 1 may lead to corner cases.